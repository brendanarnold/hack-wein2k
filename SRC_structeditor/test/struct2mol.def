3, 's.instr2mol',   'old',    'formatted',0
2, 's.dx',       'unknown',    'formatted',0
6, 's.outputstr2mol',       'unknown',    'formatted',0
20,'s.struct','old','formatted',0
