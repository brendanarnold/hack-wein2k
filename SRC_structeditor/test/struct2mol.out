0.000u 0.000s 0:00.00 0.0%	0+0k 0+0io 0pf+0w
Starting DX user interface
